src/components/Book.js

```diff
@@ -18,19 +18,23 @@ function Book({ id, title, author }) {
         <div className="book__author">{author}</div>
         <div className="book__action">
           <button type="button">Comments</button>
-          <button type="button" onClick={handleRemoveBook}>Remove</button>
+          <button type="button" onClick={handleRemoveBook}>
+            Remove
+          </button>
           <button type="button">Edit</button>
         </div>
       </div>
 
       <div className="book__progress">
-        <div className="book__progressCircle">Ø</div>
-        <div className="book__progressPercent">64%</div>
-        <div className="book__progressStatus">Completed</div>
+        <div className="book__progressCircle">o</div>
+        <div>
+          <div className="book__progressPercent">64%</div>
+          <div className="book__progressStatus">Completed</div>
+        </div>
       </div>
```

Comment:
- [ ] [REQUIRED] Great design you have here @OkoyeCharles , however, you need to ensure that you stick to the color pattern provided in the  [zeplin template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). By fixing this, you will be sticking to the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![11](https://user-images.githubusercontent.com/58692590/185496705-a9759712-307f-4820-9c02-00a886682822.png) | ![12](https://user-images.githubusercontent.com/58692590/185496730-a6b08557-453c-4a87-89db-4d191289509f.png)

Reply:
Thanks very much for the review and I will implement that immediately. Charles